mockserver/src/main/groovy/pl/touk/mockserver/server/Mock.groovy

```diff
@@ -37,6 +37,9 @@ class Mock implements Comparable<Mock> {
     Map<String, String> imports = [:]
     boolean preserveHistory = true
     Https https
+    int maxUses = -1
+    int usesLeft
```

Comment:
maybe it is better to have `used` counter and increment it always when mock is applied, then method `decrementUses` won't be necessary

Reply:
It is still necessary, but renamed to `incrementUses`. I chose this option, because it is better (in my opinion) to log uses left.

Reply:
Logging is not an answer - you can log whatever you want :)

Reply:
We talked about that and decided to keep it as it is.